Directory.Build.props

```diff
@@ -14,8 +14,9 @@
     <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
     <IsUwpProject Condition="'$(IsDesignProject)' != 'true'">$(MSBuildProjectName.Contains('Uwp'))</IsUwpProject>
     <IsSampleProject>$(MSBuildProjectName.Contains('Sample'))</IsSampleProject>
-    <DefaultTargetPlatformVersion>19041</DefaultTargetPlatformVersion>
-    <DefaultTargetPlatformMinVersion>17763</DefaultTargetPlatformMinVersion>
+    <TargetPlatformBaseVersion>10.0</TargetPlatformBaseVersion>
+    <TargetPlatformRevision>19041</TargetPlatformRevision>
```

Comment:
Is there a specific reason why you are renaming these properties? These properties names seems wrong, as they are not revisions, but big Windows release numbers. Also, I don't see a reason to have a TargetPlatformBaseVersion, as its very unlikely that it will change.

Reply:
Having those separated helps in referring them separately. You can see how I use them in #3794.